### DISCUSSION TITLE
Do rewrites in next.config.js that use the /api path not get hosted on Vercel?

### DISCUSSION THREAD

**Discussion by zachweinberg** [vercel/community#491] [Help] [answered]
Hello,
I successfully am using the `rewrites` field in next.config.js with a source of `/api/graphql`. On localhost, the proxy works, but when I push my project to Vercel's Next.js hosting, the route 404's. Is the /api base path reserved on Vercel's platform?

Thanks

**Comment by samcx** [comment, marked as answer]
@zachweinberg Is the `api/graphql` an existing API route in your app? If yes, you may have to use the `beforeFiles` property → https://nextjs.org/docs/api-reference/next.config.js/rewrites.
